src/render/FontHandler.cpp

```diff
@@ -29,17 +29,19 @@
 #include <iomanip>
 #include <sstream>
 #include <cmath>
+#include "common/common.h"
 
 #define TEXTBORDER 0.003
 
-FontHandler* FontHandler::instance = NULL;
-std::string FontHandler::m_parentPath ="";
+FontHandler *FontHandler::instance = NULL;
+std::string FontHandler::m_parentPath = std::string("fonts") + OS_SLASH;
```

Comment:
There are still lots of variables that mix snake_case and CamelCase 😢 - can you give a pass?

Reply:
I renamed `m_parentPath`. 
OS_SLASH is considered a preprocessor directive.

Reply:
Why not `m_parent_path`?